Makefile

```diff
@@ -500,6 +500,15 @@ engine_image:
 	@which docker || (echo "docker not found in ${PATH}"; exit 1)
 	./engine/test/utils/run_engine.sh build
 
+engine_image_linux: 
+	@which docker || (echo "docker not found in ${PATH}"; exit 1)
+	GOOS=linux GOARCH=amd64 $(GOBUILD) -ldflags '$(LDFLAGS)' -o bin/tiflow ./cmd/tiflow/main.go
+	docker build --platform linux/amd64 -f ./engine/deployments/docker/dev.Dockerfile -t dataflow:test ./ 
```

Comment:
Is it necessary to specific platform here

Reply:
This speeds up compilation considerably, and makes it easy to compile images available to the test_infra platform on an ARM machine.